core/src/main/java/org/infinispan/persistence/sifs/LogAppender.java

```diff
@@ -77,6 +79,10 @@ public synchronized void start(Executor executor) {
       writeProcessor.observeOn(Schedulers.from(executor))
             .subscribe(this, e -> log.warn("Exception encountered while performing write log request ", e));
 
+      completionProcessor = UnicastProcessor.create();
```

Comment:
Should we complete it once after the `writeProcessor` completes?

Reply:
Yeah, it should be completed and set to null like the other processor